src/reporters/caller-playwright.js

```diff
@@ -86,7 +86,11 @@ class CallerPlaywright extends BaseClass {
       // get the case ids from the suite
       for (const val of suite.allTests()) {
         const case_details = self.utils._formatTitle(val.title);
-        if (case_details != null) case_ids.push(parseInt(case_details[1]));
+        if (case_details != null) {
+          case_ids.push(parseInt(case_details[1]));
+        } else {
```

Comment:
what about adding `else if (${create_missing_cases}) {` ?

Reply:
Updated this functionality per your comment.